plugins/editor/src/editor/Editor.cpp

```diff
@@ -1563,6 +1579,51 @@ void Editor::Impl::updateSWLastLabel(unsigned sw, const char* label)
         updateKeyswitchNameLabel();
 }
 
+void Editor::Impl::updateBackgroundImage(const char* filepath)
+{
+    const fs::path sfzFilePath = fs::u8path(currentSfzFile_);
+    const fs::path sfzDirPath = sfzFilePath.parent_path();
+    const fs::path imagePath = sfzDirPath / filepath;
+
+    int width, height, channels;
+    image_u image {
+        stbi_load(imagePath.c_str(), &width, &height, &channels, STBI_rgb_alpha)
```

Comment:
Has problems with Unicode paths on Windows OS, because `stbi_load` calls `fopen` (which uses ANSI encoding).
On windows, prefer `_wfopen` with `imagePath.wstring().c_str()` together with `stbi_load_file`.